packages/cli/src/__tests__/commands/Version.test.ts

```diff
@@ -60,10 +60,14 @@ describe('version', () => {
 
   // Binary Tests
 
-  testIf(!useNodeAPI)('basic version', async () => {
-    const data = await ctx.cli('--version')
-    expect(cleanSnapshot(data.stdout)).toMatchSnapshot()
-  })
+  testIf(!useNodeAPI)(
+    'basic version',
+    async () => {
+      const data = await ctx.cli('--version')
+      expect(cleanSnapshot(data.stdout)).toMatchSnapshot()
+    },
+    50000,
```

Comment:
```suggestion
    10_000,
```
Probably enough, the default was 5s

Though it's weird ([Slack thread](https://prisma-company.slack.com/archives/C02A1646TBR/p1660638787096929))